src/export/markdown/index.ts

```diff
@@ -22,40 +23,6 @@ type Options = {
   titleOnlyInFrontmatter?: boolean;
 };
 
-export async function createFrontmatter(
-  session: ISession,
-  block: Block,
-  version: Version<Blocks.Article | Blocks.Notebook>,
-) {
-  const project = await new Project(session, block.id.project).get();
-  const { affiliations } = project.data;
-  const authorsData = block.data.authors ?? project.data.authors;
-  const authors = !authorsData
-    ? undefined
-    : authorsData.map((author) => {
-        const authorAffiliations = author.affiliations
-          .map((id) => affiliations.find(({ id: aid }) => id === aid)?.text || null)
-          .filter((a) => a);
-        return {
-          name: author.name || '',
-          userId: author.userId || undefined,
-          orcid: author.orcid || undefined,
-          corresponding: author.corresponding || undefined,
-          email: author.email || undefined,
-          roles: author.roles?.length > 0 ? author.roles : undefined,
-          affiliations: authorAffiliations?.length > 0 ? authorAffiliations : undefined,
-        };
-      });
-  return {
-    title: block.data.title,
-    description: block.data.description || '',
-    date: formatDate('date' in version.data ? version.data.date : version.data.date_created),
-    authors,
-    name: block.data.name,
-    oxa: oxaLink('', block.id),
-  };
-}
-
```

Comment:
ok, so this will have an immediate effect on export of `md`. I think we need to make the change to keep markdown export working as-is, by patching in the inherited frontmatter on export.